src/current/advisories/a114393.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Technical Advisory 114393
+advisory: A-114393
+summary: A bug could cause a query plan to skip scanning rows from the local region when performing a lookup join with a REGIONAL BY ROW table as the input.
+toc: true
+affected_versions: v23.1.0 to v23.1.12, and pre-release versions v23.1.0-beta.1, v23.1.0-beta.2, v23.1.0-beta.3, v23.1.0-rc.1, and v23.1.0-rc.2
+advisory_date: 2023-11-20
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+A critical bug has been identified in the [optimizer](../v23.1/cost-based-optimizer.html) that may lead to incorrect query results, affecting queries involving [REGIONAL BY ROW tables](../v23.1/regional-tables.html#regional-by-row-tables). The bug can cause rows in the local region of a REGIONAL BY ROW table to be omitted from query results when performing a [locality-optimized](../v23.1/cost-based-optimizer.html#locality-optimized-search-in-multi-region-clusters) [lookup join](../v23.1/joins.html#lookup-joins). The bug manifests under the following conditions:
```

Comment:
(ignore if this is a rush)

all capitalized mentions of REGIONAL BY ROW tables could be in code font

Reply:
formatted in code font.